vendor/github.com/envoyproxy/go-control-plane/envoy/config/filter/http/jwt_authn/v2alpha/config.pb.validate.go

```diff
@@ -0,0 +1,1020 @@
+// Code generated by protoc-gen-validate. DO NOT EDIT.
```

Comment:
Did you follow the steps in https://github.com/istio/istio/wiki/Vendor-FAQ? I think (but am not sure) there should be some go modules files changing if vendor is changing?

Reply:
The package (`github.com/envoyproxy/go-control-plane`) is already in `go.mod` and this PR uses some new types from the package that is not used before, I think that's why there is no update to `go.mod` in this PR.

I just followed the steps in the link again, it does some cleaning but not related to this PR.